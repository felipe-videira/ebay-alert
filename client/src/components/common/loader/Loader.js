import './Loader.scss';
import React from 'react'; 
import { Spin } from 'antd';

export default props => <Spin {...props} className="loader"/>