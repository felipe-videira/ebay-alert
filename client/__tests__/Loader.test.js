import React from 'react';
import Loader from '../src/components/common/loader/Loader';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const loader = renderer.create(<Loader />).toJSON();

  expect(loader).toMatchSnapshot();
});
