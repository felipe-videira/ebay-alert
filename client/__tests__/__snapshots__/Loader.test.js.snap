// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-spin ant-spin-spinning loader"
>
  <span
    className="ant-spin-dot ant-spin-dot-spin"
  >
    <i
      className="ant-spin-dot-item"
    />
    <i
      className="ant-spin-dot-item"
    />
    <i
      className="ant-spin-dot-item"
    />
    <i
      className="ant-spin-dot-item"
    />
  </span>
</div>
`;
